ansible/templates/helm_external.yaml.j2

```
@@ -1,4 +1,4 @@
 IPs:
 {% for host in groups[server_type] %}
-  - {{ hostvars[host]["ansible_" + network_interface]["ipv4"]["address"] }}
+  - {{ hostvars[host]["ansible_" + (network_interface | default(hostvars[host].ansible_default_ipv4.interface, true)) ]["ipv4"]["address"] }}
```

If `omit` is used above, then this parameter can be removed.

```suggestion
  - {{ hostvars[host]["ansible_" + (network_interface | default(hostvars[host].ansible_default_ipv4.interface)) ]["ipv4"]["address"] }}
```